test/e2e/logs_test.go

```diff
@@ -535,6 +535,13 @@ var _ = Describe("Podman logs", func() {
 		Expect(logs).To(Not(Exit(0)))
 	})
 
+	It("podman logs with non ASCII log tag succeeds", func() {
```

Comment:
You gonna need to setenv `LANG`, and please also add a defer-unsetenv.

Reply:
Oh, please also add a `LANG=C` test and confirm exit=126, stderr="Invalid byte sequence"

Taking a step back: is there any reason not to simply hardcode `C.utf8` into conmon?

Reply:
> Taking a step back: is there any reason not to simply hardcode C.utf8 into conmon?

tbh I am not sure what the correct approach should be, I just found this to work